### ISSUE TITLE
Support EC Key-Imports for the JavaKeystoreKeyProvider 

### ISSUE THREAD

**Issue by wistefan** [keycloak/keycloak#26936]
### Description

Currently, its only possible to import keys of type RSA into Keycloak, using the  [JavaKeystoreKeyProvider](https://github.com/keycloak/keycloak/blob/main/services/src/main/java/org/keycloak/keys/JavaKeystoreKeyProvider.java). In order to support the various signature methods for [OID4VC](https://github.com/keycloak/keycloak/issues/25936), it would be good to also support the import of EC keys.  

### Motivation

To properly support Verifiable Credentials and the usage of stable issuer did's, a method to import EC keys into Keycloak is required.

**Comment by tnorimat** [comment]
@wistefan As you might know, current Java Keystore Import feature of Keycloak does only support RSA key.
https://github.com/keycloak/keycloak/blob/main/services/src/main/java/org/keycloak/keys/JavaKeystoreKeyProvider.java
```
public class JavaKeystoreKeyProvider extends AbstractRsaKeyProvider {

```

**Comment by wistefan** [comment]
Yes, I noticed that. I would change the JavaKeystoreProvider to also support EC keys(and potentially EdDSA later, too). Its realtively easy, I already have a working PoC. @tnorimat what do you think?

**Comment by tnorimat** [comment]
@wistefan Thank you. I think it might be good to submit your PoC as Draft PR. Of if it is easy for you to implement the feature, please complete its implementation and submit it as a normal PR.

**Comment by tnorimat** [comment]
@wistefan I think that EdDSA support be a follow-up issue of this issue.
